esmvaltool/recipes/recipe_autoassess_landsurface_permafrost.yml

```diff
@@ -11,6 +11,9 @@ documentation:
     
   authors:
     - predoi_valeriu
+    - sellar_alistair
+
+  titile: Land-surface assessment area diagnostic that computes permafrost (from Autoassess).
```

Comment:
I suggest "computes permafrost indices", for consistency with stratosphere below. Perhaps delete "assessment area" to shorten the title.